v1/migrations/4_deploy_registry.js

```diff
@@ -0,0 +1,9 @@
+const Registry = artifacts.require("Registry");
+const { getKeysForNetwork, deployAndGet, addToTdr } = require("../../common/MigrationUtils.js");
+
+module.exports = async function(deployer, network, accounts) {
```

Comment:
While you're adding the deployment, could you copy over the deployment verification for the v0 registry from https://github.com/UMAprotocol/protocol/blob/2fb468d7c8391e9ef9893bdb9311012deb0a3308/v0/scripts/CheckDeploymentValidity.js#L19-L21 to https://github.com/UMAprotocol/protocol/blob/master/v1/scripts/CheckDeploymentValidity.js.

Reply:
Ran:
`$(npm bin)/truffle migrate --reset --network=test`
and then:
` $(npm bin)/truffle exec scripts/CheckDeploymentValidity.js --network=test
Using network 'test'.

Deployment looks good!
`